src/librustc_mir/util/pretty.rs

```diff
@@ -448,14 +448,14 @@ fn write_scope_tree(
 
         writeln!(
             w,
-            "{0:1$} // in {2}",
+            "{0:1$} // at {2}",
             indented_debug_info,
             ALIGN,
-            comment(tcx, var_debug_info.source_info),
+            tcx.sess.source_map().span_to_string(var_debug_info.source_info.span),
```

Comment:
Hmm, does it make sense to perhaps do this to locals too? Since they're also printed inside their scope.

Reply:
Or maybe neither of these changes should be in this PR? To make reviewing faster?

Reply:
ok, I'll remove it and make it separate PRs